### ISSUE TITLE
[Bug]: Unable to create rows in Airtable

### ISSUE THREAD

**Issue by AS-Laguna** [appsmithorg/appsmith#23743]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

We're unable to create new rows in Airtable. All of the other functions seem to work fine, but even hard-coding the data in the insert query doesn't work. The change isn't reflected in Airtable. There is no error, and running the query returns a result as expected. However, the result doesn't include the data we tried to insert.

### Steps To Reproduce

1. Connect to an Airtable datasource.
2. Create a new insert query.
3. Try to insert a row.
4. Open Airtable or query the table in Appsmith, and observe that the row was not added.

### Environment

Production

### Version

Cloud v1.9.20-SNAPSHOT

<img src="https://front.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_al4uk)

**Comment by sumitsum** [comment]
Analysis by Felix: https://theappsmith.slack.com/archives/C040LHZN03V/p1685039268443679

TLDR: we seem to be using `GET` request instead of `POST` when making the query execution call to the Airtable API server.
